doc/classes/Camera.xml

```diff
@@ -137,6 +137,11 @@
 		</method>
 	</methods>
 	<members>
+		<member name="affect_lod" type="bool" setter="set_affect_lod" getter="get_affect_lod" default="true">
+			If [code]true[/code], the camera will be used to calculate the level of detail in [LOD] nodes.
+			Objects further from the camera will select lower levels of detail than those closer to the camera.
+			[b]Note:[/b] Cameras with [constant PROJECTION_ORTHOGONAL] will not affect level of detail.
```

Comment:
May be good to say this property does nothing with orthogonal projection.
```suggestion
			[b]Note:[/b] This property has no effect if [member projection] is set to [constant PROJECTION_ORTHOGONAL].
```
Feel free to adjust.